README.md

```diff
@@ -122,6 +122,35 @@ Please try the following solutions:
    remote: this repository, visit https://github.com/enterprises/microsoftopensource/sso?authorization_request=AEJANEWOPPW6YTNW5TYNW2K7OBDR3A5PN5ZGOYLONF5GC5DJN5XF62LEZYAF32PCVVRXEZLEMVXHI2LBNRPWSZGODVDHWBVPMNZGKZDFNZ2GSYLML52HS4DFVNHWC5LUNBAWGY3FONZQ
    ```
 
+1. If you see the below errors while cloning the templates repository, this might be caused by the latest `Git Credential Manager Core` has been recently installed and you need to re-authorize the application.
+   ```bash
+   $ git clone https://github.com/microsoft/templates.docs.msft.git
+   Cloning into 'templates.docs.msft'...
+   remote: The `microsoft' organization has enabled or enforced SAML SSO. To access
+   remote: this repository, you must re-authorize the OAuth Application `Git Credential Manager`.
+   fatal: unable to access 'https://github.com/microsoft/templates.docs.msft.git/': The requested URL returned error: 403
+   ```
+
+   Please follow the steps below to re-authorize:
+
+   1. Go to [github settings page](https://github.com/settings/applications).
+   1. Go inside `Git Credential Manager` and click `Revoke access`.
```

Comment:
If the user signed in with the `PAT`, they won't find the `Git credential manager` in the applications.